sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -994,19 +998,23 @@ class BlobSasPermissions(object):
         destination of a copy operation within the same account.
     :param bool delete:
         Delete the blob.
+    :param bool delete_version:
+        Delete the blob version for the versioning enabled storage account.
```

Comment:
I'm confused by this.... does it mean we are giving permission to delete non-current blob versions? Or any blob in a version-enabled-account?

Reply:
Yes, it's to give permission to delete the non-current(past) versions

Reply:
Then I think we should rename this... maybe delete_previous_versions or something...